includes/amp-post-template-functions.php

```diff
@@ -54,6 +55,20 @@ function amp_post_template_add_fonts( $amp_template ) {
 	}
 }
 
+/**
+ * Add block styles for core blocks and third-party blocks.
+ *
+ * @param AMP_Post_Template $amp_template Template.
+ */
+function amp_post_template_add_block_styles( $amp_template ) {
+	global $wp_styles;
+
+	current_theme_supports( 'wp-block-styles' );
```

Comment:
```suggestion
	add_theme_supports( 'wp-block-styles' );
```

Reply:
🤦‍♂ 

Yesterday was definitely not my brightest day...